index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="http://spiermar.github.io/d3-flame-graph/d3.flameGraph.css">
+  </head>
+  <body>
+    <div class="container">
+      <div id="chart">
+      </div>
+    </div>
+
+    <script type="text/javascript" src="http://d3js.org/d3.v3.min.js"></script>

```

Comment:
I was not able to find flame graph library integrated with d3 v4. 
As you can see from https://github.com/spiermar/d3-flame-graph/blob/master/package.json
it depends on "d3": "^3.5.6"